wallet-new/src/Cardano/Wallet/API/V1/Accounts.hs

```diff
@@ -37,4 +40,14 @@ type API
           :> "certificates"
           :> ReqBody '[ValidJSON] Redemption
           :> Post '[ValidJSON] (WalletResponse Transaction)
+    :<|> "wallets" :> CaptureWalletId :> "accounts"
+          :> CaptureAccountId :> "addresses"
+          :> Summary "Retrieve only account's addressees."
```

Comment:
_addresses_ ? 🤔

Reply:
Referring to the typo _addressees_ here ?

Reply:
yup 😃